.github/workflows/ci.yml

```diff
@@ -95,9 +98,29 @@ jobs:
           bpkg build libodb-sqlite --yes
           bpkg build libodb-pgsql --yes
           bpkg install --all --recursive
-          echo "CMAKE_PREFIX_PATH=$HOME/odb_install:$CMAKE_PREFIX_PATH" >> $GITHUB_ENV
-          echo "PATH=$HOME/odb_install/bin:$PATH" >> $GITHUB_ENV
 
+      - name: Export environment variables Ubuntu 18
+        if: ${{ matrix.os == 'ubuntu-18.04' }}
+        run: |
+          ls $HOME/odb_install/bin
+          ls $HOME/thrift_install/bin
+          echo "CMAKE_PREFIX_PATH=$HOME/thrift_install:$HOME/odb_install:$CMAKE_PREFIX_PATH" >> $GITHUB_ENV
+          echo "PATH=$HOME/thrift_install/bin:$HOME/odb_install/bin:$PATH" >> $GITHUB_ENV
+
+      # Ubuntu 20.04 commands
+      - name: Install required packages Ubuntu 20
+        if: ${{ matrix.os == 'ubuntu-20.04' }}
+        run: sudo apt-get install -y git cmake make g++ libboost-all-dev llvm-10-dev clang-10 libclang-10-dev odb libodb-dev thrift-compiler libthrift-dev default-jdk libssl-dev libgraphviz-dev libmagic-dev libgit2-dev ctags libgtest-dev npm
```

Comment:
Probably split long lines into multiple lines for easier readability:
https://stackoverflow.com/questions/59954185/github-action-split-long-command-into-multiple-lines

Reply:
I tried this solution from time to time, it never worked. I tried it yesterday as well, still no good results. Instead I chose to use the basic yaml syntax to split lines. (>)